wmn-data.json

```diff
@@ -1970,7 +1970,7 @@
        },
        {
         "name" : "eToro",
-        "uri_check" : "https://www.etoro.com/people/{account}",
+        "uri_check" : "https://www.etoro.com/api/logininfo/v1.1/users/{account}",

```

Comment:
Is there a `pretty_uri` you can add here that has a direct link to the user's profile?

Reply:
yep

Reply:
@WebBreacher : done ☑️